Changelog.md

```diff
@@ -4,6 +4,7 @@ Language Features:
 
 
 Compiler Features:
+ * Drop obsolete experimental support for EIP-615, resp. for the ``--machine evm15`` in the Command Line Interface.
```

Comment:
obsolete? Is it really EIP-615 or just "EVM15"?

Reply:
I can drop "obsolete" - but the PR drops the EIP-615 instructions and their use as the evm15 machine, so not sure what you mean by "Is it really EIP-615 or just "EVM15"?"?

Reply:
I now changed it to just say, it drops ``--machine evm15``.